File: 0268-missing-number/0268-missing-number.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var missingNumber = function (nums) {
    let n = nums.length;
    let sum = 0
    for (let i = 0; i < nums.length; i++) {
        sum += nums[i]
    }
    let sumOfFirstNumbers = n * (n + 1) / 2;
    return sumOfFirstNumbers - sum;
};

File: 0268-missing-number/0268-missing-number.py
class Solution(object):
    def missingNumber(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        n = len(nums)
        sum = 0
        for i in range(len(nums)):
            sum += nums[i]
        sumOfFirstNumbers = n*(n+1)/2
        return sumOfFirstNumbers - sum